x-pack/plugins/alerting/server/alert/alert.test.ts

```diff
@@ -695,6 +695,7 @@ describe('isFilteredOut', () => {
           _id: '1',
           _index: '.alerts',
           '@timestamp': '',
+          // @ts-expect-error
```

Comment:
For summarized alerts, we are still reading the alert doc and unflattening. This matches what detection rules were doing when they generated `context.alerts` (reading flattened alerts and unflattening for notifications). This `// @ts-expect-error` is because our schema is now expecting flattened keys.

Reply:
Gotcha, thanks!